Detectors/TPC/base/include/TPCBase/Sector.h

```diff
@@ -26,17 +26,15 @@ namespace TPC {
 class Sector
 {
   public:
-    enum
-    {
-        MaxSector = 36
-    };
+    // the number of sectors
+    static constexpr char MAXSECTOR=36;
```

Comment:
An enumeration is more for a set of well defined things. In this case, we really only want to define a constant (so I changed also according to coding conventions).